src/main/java/no/nav/tag/finnkandidatapi/unleash/FeatureToggleService.java

```diff
@@ -0,0 +1,29 @@
+package no.nav.tag.finnkandidatapi.unleash;
+
+import lombok.RequiredArgsConstructor;
+import no.finn.unleash.Unleash;
+import no.finn.unleash.UnleashContext;
+import no.nav.tag.finnkandidatapi.tilgangskontroll.TokenUtils;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class FeatureToggleService {
+
+    private final Unleash unleash;
+    private final TokenUtils tokenUtils;
+
+    public boolean isEnabled(String feature) {
+        return unleash.isEnabled(feature);
+    }
+
+    public boolean isEnabledMedPilotkontor(String feature) {
+        return unleash.isEnabled(feature, contextMedInnloggetBruker());
+    }
```

Comment:
Jeg synes ikke vi bør gjøre forskjell på 'isEnabled' og 'isEnabledMedPilotKontor'.
Det å sette inn innlogget bruker på contexten er ikke direkte relatert til pilotkontor, men det gjør det mulig å benytte bruker-relaterte toggle-strategier. (ikke bare _byEnhet_-strategien for pilotkontor, men også _userWithId_ og _gradualRolloutUserId_). Det å ha ulike _isEnabled_-metoder der kun den ene gjør det mulig å bruke brukerrelaterte toggle-strategier blir bare vanskelig. Effekten er at dersom du har en toggle _er.denne.aktiv_, og bruker strategien byEnhet (eller withUserId), vil den kunne returnere _true_ dersom den evalueres med `isEnabledMedPilotkontor`, men _false_ dersom den evalueres med `isEnabled`, og det gir lite mening.
Altså: Dropp `isEnabledMedPilotkontor` og la `isEnabled` alltid opprette unleashContext-en med userId :)

Reply:
Grunnen til at jeg lagde to `isEnabled()` var at man også bruker unleash i den ene kafkaprodusenten, hvor man ikke har noe forholdt til hvem som er logga inn. Hva setter man userId til da?